tooltester/README.md

```diff
@@ -1,80 +1,53 @@
 Last tested with ansible 2.2.1.0
+
+The general idea is to only have the `jenkins` user talk to each other, never using the `ubuntu` user
+
 # Master Setup:
-1.  Install the Jenkins pipeline suite
-2.  Install these plugins:
-- Blue Ocean 1.0.1
```

Comment:
Was this content moved (can't see where it ended up)

Reply:
All the plugins were installed by default using the jenkinsci/blueocean docker image.  the only one that didn't is ansible v1.0 plugin